src/Exception/DispatchingException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Equip\Exception;
+
+use InvalidArgumentException;
+
+class DispatchingException extends InvalidArgumentException
+{
+    /**
+     * @param string $spec
+     *
+     * @return static
+     */
+    public static function notInvokable($spec)
+    {
+        return new static(sprintf(
+            'Dispatcher `%s` is not invokable',
+            $spec
```

Comment:
Assuming `$spec` is `callable`, I think this would result in an error when `sprintf` attempts to cast it to a string, e.g.

```
PHP Catchable fatal error:  Object of class Closure could not be converted to string in Command line code on line 1
```